app/models/user.rb

```diff
@@ -11,7 +11,6 @@ class User < ApplicationRecord
   has_many :viewing_parties
 
   def password_complexity
-    # Regexp extracted from https://stackoverflow.com/questions/19605150/regex-for-password-must-contain-at-least-eight-characters-at-least-one-number-a
```

Comment:
I would leave this in here as it is almost credit for where the code came from. Or an explanation of it.